x11-misc/synergy_haiku/synergy_haiku-0.3.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Mouse and Keyboard sharing software"
+DESCRIPTION="Synergy brings your computers together in one cohesive experience; its free \
+and open source cross-platform software for sharing one mouse and keyboard \
+between multiple computers on your desk. It works on Windows, Mac OS, Linux \
+and Haiku allowing you to seamlessly move your mouse cursor between computers. \
+
+Credits for: Axel Dörfler (ATKeymap) - Stefano Ceccherini (wrapper) - Jérôme Duval (Keymap) \
+Alex Evans (uSynergy) - Jessica Hamilton (wrapper) - Ed Robbins (wrapper) \
+
+Usage: Create a configuration file at ~/config/settings/synergy with:
+	enable = true
+	server = 192.168.1.X (ip of your server)
+	enableCLipboard = false (false recomended)
+
+For more info and options in config file visit: https://github.com/unarix/synergy_haiku
+"
+HOMEPAGE="https://symless.com/synergy/"
+COPYRIGHT="2020 - Nahuel Tello (Fix & Package) - Axel Dörfler (ATKeymap) - Stefano Ceccherini (wrapper) \
+- Jérôme Duval (Keymap) - Alex Evans (uSynergy) - Jessica Hamilton (wrapper) - Ed Robbins (wrapper)"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/unarix/synergy-haiku/archive/0.3.tar.gz"
+CHECKSUM_SHA256="d76d218cb1df6740a71d3aa02ecd76860ccdfb644cb0c11a85d472e1d4435550"
+SOURCE_DIR="synergy_haiku-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	synergy_haiku$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	makefile_engine
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	echo "Execute make ..."
```

Comment:
```suggestion
```

We don't use these kind of `echo` in our recipe unless needed to ease debugging (ie. when calling `make` in a loop).